example/src/controllers/examples/subsections/FormData.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+
+/**
+ * A component used to display the underlying data model of a react-formable™ form
+ * (requires highlight.js)
+ */
+export default class FormData extends React.Component {
```

Comment:
@kaw2k I saw that [highlight.js](https://highlightjs.org/) was imported globally on the `index.html` so went down this route... is that used by `<MD />`?